code_sender/sender.py

```diff
@@ -225,15 +227,17 @@ def send_to_cmder(self, cmd):
                 send_to_cmder(cmd, conemuc)
 
     def send_to_linux_terminal(self, cmd):
+        window_name = self.settings.get("window_name")
         linux_terminal = self.settings.get("linux_terminal")
+        wids = get_linux_wids(window_name, linux_terminal)
 
         if len(re.findall("\n", cmd)) > 0:
             if self.bracketed_paste_mode:
-                send_to_linux_terminal(linux_terminal, [cmd, ""])
+                send_to_linux_terminal(wids, [cmd, ""])
```

Comment:
I would rather pass `linux_terminal` and `window_name` to `send_to_linux_terminal` and run `get_linux_wids` from there.

Reply:
Great suggestion! Done in the latest commits.